tests/pytests/data/isds/cassiniiss_isis_isd.json

```diff
@@ -0,0 +1,201 @@
+{
```

Comment:
I had to add a separate ISD for the ISIS label because the Cassini kernels set has been updated since the pds3 data was generated so values have shifted a little bit.